api/server/structs/endpoints_blob.go

```diff
@@ -12,3 +12,12 @@ type Sidecar struct {
 	KzgProof                 string                   `json:"kzg_proof"`
 	CommitmentInclusionProof []string                 `json:"kzg_commitment_inclusion_proof"`
 }
+
+type BlobSidecars struct {
+	Sidecars []*Sidecar `json:"sidecars"`
+}
+
+type PublishBlobsRequest struct {
+	BlobSidecars *BlobSidecars `json:"blob_sidecars"`
+	BeaconRoot   string        `json:"beacon_root"`
```

Comment:
This should be called either `block_root` or `beacon_block_root` since there is no such thing as a beacon root

Reply:
Updated